bridge/client/app/_components/ktb-stage-overview/ktb-stage-overview.component.ts

```diff
@@ -52,6 +59,17 @@ export class KtbStageOverviewComponent implements OnDestroy, OnInit {
     });
   }
 
+  public ngAfterContentInit(): void {
+    const stageName = this.route.snapshot.paramMap.get('stageName');
+    if (stageName && this.project) {
+      const stage = this.project.getStage(stageName);
+      if (stage) {
+        this.selectedStage = stage;
+        this.selectedStageChange.emit({ stage: stage, filterType: undefined });
+      }
+    }
```

Comment:
This is still kind of hacky... I'd rather go with you through my ideas. Will contact you.